### DISCUSSION TITLE
RecursionField在FormDialog中使用时，RecursionField的schema内的字段reactions无法找到$deps

### DISCUSSION THREAD

**Discussion by BooLeei** [alibaba/formily#3281] [Q&A 2.x] [answered]
问题描述：RecursionField在FormDialog中使用时，RecursionField的schema内的字段无法找到$deps
sandbox：
- 基于RecursionField的示例：https://codesandbox.io/s/wonderful-cohen-19r3sd?file=/App.tsx
- 基于antd FormDialog的示例：https://codesandbox.io/s/admiring-roman-ogyhpx?file=/App.tsx

**Comment by janryWang** [comment, marked as answer]
加个 x-compile-omitted绕过外部编译x-component-props，走组件内编译 https://codesandbox.io/s/suspicious-payne-eerzdq

**Reply by BooLeei** [reply]
可以了 大佬给力
